package.json

```diff
@@ -49,7 +49,8 @@
     "husky": "^0.10.2",
     "nyc": "^5.0.1",
     "snazzy": "^2.0.1",
-    "standard": "^6.0.4"
+    "standard": "^6.0.4",
+    "sugarss": "^0.1.2"
```

Comment:
i'm not too keen on having this as a devdep, but i kind of what the test to cover this use case because we'll be using it so often. we could also have `package.json` in our fixtures later on, but this works for now